android/expoview/src/main/java/versioned/host/exp/exponent/ExponentPackage.java

```diff
@@ -184,6 +186,7 @@ public static ExponentPackage kernelExponentPackage(Context context, JSONObject
         nativeModules.add(new RNCWebViewModule(reactContext));
         nativeModules.add(new NetInfoModule(reactContext));
         nativeModules.add(new RNSharedElementModule(reactContext));
+        nativeModules.add(new ExpoAppearanceModule(reactContext));
```

Comment:
Maybe let's use `ExpoAppearancePackage` here since we override `createNativeModules` there?

Reply:
It's not possible to use `ExpoAppearancePackage` without previous casting and moreover I explicitly add `ExpoAppearancePackage` in respective part (please see `public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {`.